### ISSUE TITLE
RDCMan feature request: the ability to add groups in a file, with existing entries

### ISSUE THREAD

**Issue by enachi** [MicrosoftDocs/sysinternals#619]
Please, add the ability to create groups/smart groups after adding servers. 

I have added a lot of az vm's to a rdc file and then I realized I need to group them by purpose, but the option is grayed.
For a new file, without anything added to it, that option is available but as soon as you add a server, that option is not available anymore.

other option would be to have the group membership added to some tab under the properties of an entry.. 

Thank you for this amazing software!

**Comment by kpr0th** [comment]
Yes please!

**Comment by BGleasonGH** [comment]
Yes, this would help so much with organization. System roles change, ideologies change, and the ability to add a nested group to an already existing group with server resources would help a lot! It would be interesting to understand the challenge that forces this limitation if that is the case, or if this would just a design preference.

You can move everything out to a temp group, and then create the nested group/s you want and then move EVERYTHING back, however that is impractical specially with large quantities of objects.
